packages/graphback-cli/src/utils/generateHelper.ts

```diff
@@ -0,0 +1,71 @@
+import chalk from 'chalk';
+import { accessSync, readFileSync, writeFileSync } from 'fs';
+import { GlobSync } from 'glob'
+import { GraphQLBackendCreator, IGraphQLBackend, KnexResolverManager, PostgresSchemaManager } from 'graphback-core'
+import { logError, logInfo } from '.';
+
+/**
+ * Message after command execution
+ */
+function postCommandMessage(): void {
+  logInfo(`
+Successfully generated schema and resolvers :tada:.  
+
+Next steps:
+1. Run ${chalk.cyan(`graphback db`)} to create postgres database.
+2. Run ${chalk.cyan(`docker-compose up -d`)} followed by ${chalk.cyan(`npm start`)}
+`)
+}
+
+/**
+ * Generate schema and resolvers using graphback-core and
+ * write them into generated folder
+ */
+async function generateBackend(): Promise<void> {
+  try{
+    accessSync(`${process.cwd()}/model`)
+    const model = new GlobSync('model/*.graphql', { cwd: process.cwd()})
+    
+    if(model.found.length === 0) {
+      logError(`No graphql file found inside model.`)
+      process.exit(0)
+    }
+
+    const path = `${process.cwd()}/${model.found[0]}`
+    const schemaText = readFileSync(path, 'utf8')
+
+    const outputSchemaPath = `${process.cwd()}/generated/schema.graphql`
+    const outputResolverPath = `${process.cwd()}/generated/resolvers.ts`
+    
+    const backend = new GraphQLBackendCreator(schemaText)
+    
+    const connectionConfig = {
+      'user': 'postgresql',
+      'password': 'postgres',
+      'database': 'users',
+      'host': '127.0.0.1',
+      'port': '5432'
+    }
+    
+    const manager = new PostgresSchemaManager(connectionConfig);
```

Comment:
So we do db in generate command?
Would it be possible to introduce questions for this command:

- What database you want to choose
 x Postgress (just for now but there willk be more options in future)

I would love also to not generate db here (separate db command), but if you like idea if the generate doing all then let's remove db command. 

We will need to remember user choices inside the template. 
IMHO template can have something like graphback.json that will contain information about the database connection and any other choices

Reply:
We have separated generate and db. You might not have rebuild the graphback package why it's creating db too.

Reply:
I think separation is better since db command will be pretty hefty with implementation.